### ISSUE TITLE
当我的数据集只有一个类别时，模型不收敛

### ISSUE THREAD

**Issue by zhangming8** [TencentYoutuResearch/ObjectDetection-OneStageDet#61]
当我的数据集只有一个类别时，模型不收敛？请问有遇到这个问题吗

**Comment by jacksonly** [comment]
when you modify the class_num to 1, you should modify the loss function. Because Cross Entropy is not working on binary classification.

**Comment by shenghsiaowong** [comment]
我也是只有一个类别，在控制台上打印class的概率总是0

**Comment by shenghsiaowong** [comment]
> 当我的数据集只有一个类别时，模型不收敛？请问有遇到这个问题吗

do you have solved it?
